vendor/priors/gwt.js

```diff
@@ -526,6 +518,7 @@ var batStoreCache;
 var accountStoreCache;
 var pkiStoreCache;
 var rootKeyCache;
+let SAFARI_CACHE_SIZE = 1024 * 1024 * 200;
```

Comment:
Is this the default safari gives you? If you try to exceed it does it ask the user for permission?

Reply:
I would suggest setting this to whatever safari gives you by default without warning.